docker-traffic-gen/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "scripts": {
```

Comment:
Interesting - how does one kick this off via npm? And can you please update the README to show how to use?

Reply:
`npm run <scriptname>` so `npm run start` for example. I almost made a makefile instead, but since we use npm other places ...

Will update README